### PR TITLE
Add campaign-palettes to three missions

### PR DESCRIPTION
**Pull request by Smittytron**
I noticed some of the missions I am working on had the wrong rules copy-paste. I double checked the recently added missions and sure enough, three of them were missing campaign-palettes.

### CODE REVIEW

**Comment by Smittytron** [comment]
Updated. Fall of Greece 1 had a particular problem with a green Greece. There are additional missions that use 'Allies' as the player instead of Greece so I've added them to the campaign-palettes.

File: mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua [hunk 1]
```diff
@@ -489,7 +489,7 @@ WorldLoaded = function()
 	Allies = Player.GetPlayer("Allies")
 	USSR = Player.GetPlayer("USSR")
 	BadGuy = Player.GetPlayer("BadGuy")
-	Greece = Player.GetPlayer("Greece")
+	Greece = Player.GetPlayer("England")
```
Comment (pchote) [inline mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua]: This looks more than a little bogus. Is it still needed now that you're modifying campaign-palettes?

**Review by reaperrr — APPROVED** [review]

File: mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua (hunk 1 above)
Comment (Mailaender) [reply mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua]: I tend to agree. Assuming this is an automatically imported mission you should keep the country `Greece` and just overwrite its color in .yaml if that is what you are after.

Comment (pchote) [reply mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua]: I think @Smittytron found that England was the correct player in the imported mission, and that `Greece` was a copy/paste error. Not 100% sure on that though.

Comment (Smittytron) [reply mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua]: The original had the green team as England. I didn't import the file myself but took it from a [submitter on forums](https://forum.openra.net/viewtopic.php?f=83&t=20812&p=309462&hilit=counterstrike#p309462), who changed Greece to Allies in multiple missions, and England to Greece in this particular one.

Allies made more sense to me than Greece so I kept it in several missions that I did later; not understanding palettes at the time. The options to fix this are to either add Allies to the palettes, or to change Allies to Greece in several missions.

Comment (pchote) [reply mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua]: ~Why not rename the `Greece` variable to `England`? Surely this is the most correct option.~ Already done

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/7919fe95bec8258058f4a20605c578864c9034e9...cbf216aa8b10e49809dd95055f2c82ebb3bb88de)

**Pull request merged**